doc/galgebra_guide.rst

```diff
@@ -4,7 +4,11 @@ What is Geometric Algebra?
 .. raw:: html
 
    <script type="text/x-mathjax-config">
-   MathJax.Hub.Config({TeX: { equationNumbers: { autoNumber: "AMS" } }});
+   window.MathJax = {
+     tex: {
+       tags: 'ams'
```

Comment:
So this fixed equation numbering?

Reply:
I don't know yet :)

Reply:
If my memory serves me correctly, this still won't fix it

Reply:
This did not fix it :(